marbot-aws-account-connection.yml

```diff
@@ -139,6 +146,83 @@ Resources:
           Condition:
             StringEquals:
               'sts:ExternalId': !Select [2, !Split ['/', !Ref 'AWS::StackId']]
+  MonitoringAssistantPolicy:
+    Condition: MonitoringAssistantEnabled
+    Type: 'AWS::IAM::Policy'
+    Properties:
+      PolicyDocument:
+        Statement:
+        - Effect: Allow
+          Action:
+          - 'autoscaling:DescribeAutoScalingGroups'
+          - 'cloudwatch:DeleteAlarms'
+          - 'cloudwatch:DescribeAlarms'
+          - 'cloudwatch:ListTagsForResource'
+          - 'cloudwatch:PutMetricAlarm'
+          - 'ec2:DescribeAccountAttributes'
+          - 'ec2:DescribeInstanceAttribute'
+          - 'ec2:DescribeInstanceCreditSpecifications'
+          - 'ec2:DescribeInstances'
+          - 'ec2:DescribeInstanceTypes'
+          - 'ec2:DescribeVolumes'
+          - 'ec2:DescribeVpcs'
+          - 'elasticloadbalancing:DescribeLoadBalancers'
+          - 'elasticloadbalancing:DescribeTargetGroups'
+          - 'events:DeleteRule'
+          - 'events:DescribeRule'
+          - 'events:ListTagsForResource'
+          - 'events:ListTargetsByRule'
+          - 'events:PutRule'
+          - 'events:PutTargets'
+          - 'events:RemoveTargets'
+          - 'iam:CreateRole'
+          - 'iam:DeleteRole'
+          - 'iam:DeleteRolePolicy'
+          - 'iam:GetRole'
+          - 'iam:GetRolePolicy'
+          - 'iam:ListAttachedRolePolicies'
+          - 'iam:ListInstanceProfilesForRole'
+          - 'iam:ListRolePolicies'
+          - 'iam:PassRole'
+          - 'iam:PutRolePolicy'
+          - 'iam:UpdateRoleDescription'
+          - 'lambda:AddPermission'
```

Comment:
Could we restrict the `lambda:` permissions to functions with a specific name prefix?

Reply:
done